src/components/llama-select-fab.ts

```diff
@@ -49,13 +37,16 @@ export class LlamaSelectFab extends LitElement {
 
   render() {
     return html`
-      <input type="file" accept="image/*"
-             id="select" hidden
-             @change=${this.handleInputChange} />
-      <label id="root" for="select">
-        <mwc-fab icon="file_upload"
-                 label="Select a file to store in Llaminator">
-        </mwc-fab>
-      </label>`;
+      <div class="mdc-touch-target-wrapper llama-upload-button">
```

Comment:
I don't think llama-upload-button is used

Reply:
You're right, we don't need that anymore. Deleted.